addon/package.json

```diff
@@ -13,7 +13,8 @@
   "author": "Mozilla (https://mozilla.org/)",
   "engines": {
     "firefox": ">=38.0a1",
-    "fennec": ">=38.0a1"
+    "fennec": ">=38.0a1",
+    "node" : ">=0.4.0"
```

Comment:
Looks like Circle CI tests failed on this - I don't think that's a valid engines entry for the addon